langevals/evaluators/topic_clustering/langevals_topic_clustering/incremental_clustering.py

```
@@ -0,0 +1,319 @@
+
+
+def get_matching_topic(trace: TraceWithEmbeddings, topics: list[U]) -> Optional[U]:
+    trace_embeddings = np.array(trace["embeddings"])
+    centroid_distances = cdist(
+        [trace_embeddings],
+        np.array([t["centroid"] for t in topics]),
+        "cosine",
+    ).flatten()
+
+    # Find the closest topic to assign the trace
+    sorted_topics = sorted(zip(topics, centroid_distances), key=lambda x: x[1])
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)